### PR TITLE
Sort algorithm updates

### PR DESCRIPTION
**Pull request by Genyus**
Implements solution for quick sort excercise, removes merge sort exercise

### CODE REVIEW

File: exercises/algorithms/README.md [hunk 1]
```diff
@@ -12,72 +12,55 @@ For this assignment, we've provided the starter project above.
 
 ## :pushpin: Sorting
 
-### Merge Sort<sup>1</sup>
```
Comment (tanyapowell) [inline exercises/algorithms/README.md:15]: Why is Merge Sort being removed?

File: exercises/algorithms/src/main/java/com/cbfacademy/App.java [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+
+        ArraySorter sorter = new QuickSort();
+
+        sorter.sort(arr, 0, arr.length - 1);
+
+        System.out.println("Sorted array:");
+
+        for (int num : arr) {
+            System.out.print(num + " ");
+        }
```
Comment (tanyapowell) [inline exercises/algorithms/src/main/java/com/cbfacademy/App.java]: Are all algorithm exercises expected to be run from here? If yes, would it be easier if there was some indication that this is all for the Quick Sort exercise?

**Review by sohbaker — APPROVED** [review]

File: exercises/algorithms/README.md (hunk 1 above)
Comment (Genyus) [reply exercises/algorithms/README.md:15]: For the same reason that linear search is being removed — there's just too much for them to do in a single session. We're cutting the exercises from 4 to 2

**Comment by Genyus** [comment]
Relocated sort-related files for separation from search starter files

File: exercises/algorithms/README.md (hunk 1 above)
Comment (tanyapowell) [reply exercises/algorithms/README.md:15]: Cool, have the slides been updated to reflect this?

**Review by tanyapowell — APPROVED** [review]

File: exercises/algorithms/README.md (hunk 1 above)
Comment (Genyus) [reply exercises/algorithms/README.md:15]: Yep, we'd already skipped these slides for the current cohort

**Pull request merged**